astropy/io/votable/tests/test_table.py

```diff
@@ -261,6 +263,82 @@ def test_write_with_format():
     assert b"TABLEDATA" not in obuff
 
 
+@pytest.mark.skipif(not HAS_PYARROW, reason="requires pyarrow")
+@pytest.mark.parametrize("overwrite", [True, False])
+def test_read_write_votable_parquet(tmp_path, overwrite):
+    """
+    Test to write and read VOTable with Parquet serialization
+    """
+
+    # Create some fake data
+    number_of_objects = 10
+    ids = [f"COSMOS_{ii:03g}" for ii in range(number_of_objects)]
+    redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+    mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+    sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
```

Comment:
Is randomization really necessary here? You didn't set a seed, which means you don't even check if it is reproducible. Might as well use dummy values like ones or zeroes?

Reply:
It doesn't really matter for this test whether it's the same between two different runs, and it's self consistent within itself, so I would just leave it be for now.